Version 1.2.1/startup

```diff
@@ -1,7 +1,11 @@
 term.clear()
 term.setCursorPos(1, 1)
-shell.run("delete","install")
-shell.run("delete","eject")
-shell.run("cp","disk/install","install")
-shell.run("cp","disk/eject","eject")
+if fs.exsist("/install") then
+  fs.delete("/install")
+end
+if fs.exists("/eject") then
+  fs.delete("/eject")
+end
+fs.move("/disk/install","/install")
+fs.move("/disk/eject","/eject")

```

Comment:
These 2 Lines should be `fs.copy` instead of `fs.move`

Reply:
Fair. I've been out of the loop on CC in a while, so thanks 😄